services/QuillLMS/spec/integration/comprehension_integration_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'csv'
+require 'httparty'
+
+describe 'ComprehensionIntegration' do
+  FEEDBACK_URL = 'https://comprehension-feedback.quill.org'
+  SESSION_ID = 'INTEGRATION_TEST'
+
+  # CSV generated via the following query
+  # SELECT prompt_id, c_p.text AS stem, entry, f_h.optimal, c_r.rule_type, rule_uid, f_h.concept_uid
```

Comment:
Maybe move this query to Metabase (and link to it), both to clear up this file and to make csv generation easy.

Reply:
Good call.  Updated.